### ISSUE TITLE
aria-labelledby has invalid value

### ISSUE THREAD

**Issue by ccveer** [downshift-js/downshift#1190]
- `downshift` version: 6.0.6
- `node` version: 10.13.0
- `npm` version: 6.4.1

**Reproduction Steps**:
Any implementation of Downshift without a label

**Problem description**:
Downshift adds an aria-labelledby property by default, but that is an invalid id when there is no label and is therefore an a11y bug.  I do not need to add an aria descriptor for my usage of Downshift, but I can't have invalid aria attributes either.
![image](https://user-images.githubusercontent.com/49158600/96900293-8f128d00-1457-11eb-92e4-139654644891.png)

**Suggested solution**:
Allow an override of the aria-labelledby property or remove the default value.

**Comment by silviuaavram** [comment]
You can override it. If you are using useCombobox `{...getInputProps({'aria-labelledby': null})}`.

**Comment by silviuaavram** [comment]
We strongly support physical labels. If you don't support one, you can use the solution above and `aria-label` instead.
